fixes thread debugging being available
by default without setting any breakpoints or usage of libthread_db.

--- gdb-7.4.org/gdb/linux-nat.c	2013-08-10 05:24:24.651000003 +0000
+++ gdb-7.4/gdb/linux-nat.c	2013-08-10 05:25:50.966000003 +0000
@@ -2265,7 +2273,7 @@
 		status = 0;
 	    }
 
-	  if (non_stop)
+	  if (1)
 	    {
 	      /* Add the new thread to GDB's lists as soon as possible
 		 so that:
